### ISSUE TITLE
Add onHover property to navbar

### ISSUE THREAD

**Issue by zeim839** [ufosc/Club_Website_2#23]
**Todo:** add an on-hover property to the navbar that highlights the bottom of navigation buttons with green.
**Preview:**
![demo](https://user-images.githubusercontent.com/50573884/195960890-86dbaa57-13b5-497c-95ba-7c6dde218564.png)

When the user moves the mouse over (e.g) the "About" button, a green bar should appear directly below it. Note that this is for hovering only, current pages will not be highlighted. 

Color: Hex #76F936

The navbar is defined in:
```
views/components/navbar.ejs
public/css/navbar.css
public/css/common.css
```

**Comment by willfpearce** [comment]
made pr :)

**Comment by zeim839** [comment]
Implemented in #28
